include/ListView/ListViewGrid.php

```diff
@@ -46,20 +46,16 @@ public function Show() {
 		$viewid = isset($_SESSION['lvs'][$this->module]) ? $_SESSION['lvs'][$this->module]['viewname'] : 0;
 		$focus = new $this->module();
 		$focus->initSortbyField($this->module);
-		$gvOrderField = GlobalVariable::getVariable('Application_ListView_Default_OrderField', '');
-		$gvSortOrder = GlobalVariable::getVariable('Application_ListView_Default_Sort_Order', '');
-		$gvDefaultSorting = GlobalVariable::getVariable('Application_ListView_Default_Sorting', false);
+		$sortArrayList = $focus->getOrderByAndSortOrderList();
 		if (isset($_REQUEST['sortAscending'])) {
 			$this->orderBy = $_REQUEST['sortAscending'] == 'true' ? 'ASC' : 'DESC';
-		} elseif (!empty($gvSortOrder) && $gvDefaultSorting) {
-			$this->orderBy = $gvSortOrder;
+		} else {
+			$this->orderBy = count($sortArrayList) ? $sortArrayList[0]['sortOrder'] : '';
```

Comment:
empty is faster than count, change to

`$this->orderBy = empty($sortArrayList) ? '' : $sortArrayList[0]['sortOrder'];`